pkg/oc/clusterup/coreinstall/kubeapiserver/openshift_controller.go

```diff
@@ -15,22 +17,15 @@ func MakeOpenShiftControllerConfig(existingMasterConfig string, basedir string)
 	if err := tmpformac.CopyDirectory(existingMasterConfig, configDir); err != nil {
 		return "", err
 	}
-
-	// update some listen information to include starting the DNS server
-	masterconfigFilename := path.Join(configDir, "master-config.yaml")
-	masterconfig, err := componentinstall.ReadMasterConfig(masterconfigFilename)
-	if err != nil {
+	if err := os.Remove(path.Join(configDir, "master-config.yaml")); err != nil {
```

Comment:
why? to migrate ?

Reply:
> why? to migrate ?

to make sure nothing depends on it by accident.